### PR TITLE
Created the navigation for mobile & desktop

### LINKED ISSUE #72: Building navigation for mobile & laptop

### PR DESCRIPTION
**Pull request by veerleprins**
Let me know if I need to change something 👍🏼 !

Screenshot 2021-06-15 at 14 40 30
Screenshot 2021-06-15 at 14 41 05
Screenshot 2021-06-15 at 14 41 12

### CODE REVIEW

**Review by SharonV33 — APPROVED** [review]
looks good! nice going

File: src/components/atoms/List.svelte [hunk 1]
```diff
@@ -2,18 +2,45 @@
...
+    }
+    &.active {
+      font-family: $bold-font;
+      border-bottom: 1px solid $ui-blue;
+    }
+  }
+  a {
+    color: $ui-black;
     text-decoration: none;
+    // &:visited {
```
Comment (johancvb) [inline src/components/atoms/List.svelte:36]: Klopt het dat er nog comments in de code horen te zitten?

Comment (veerleprins) [reply src/components/atoms/List.svelte:36]: Yess, this is for the slider (today, yesterday etc.) that still needs to be adapted for mobile and desktop! It has nothing to do with the navigation in this case :)

**Review by johancvb — APPROVED** [review]
Very nice work! 👑

**Pull request merged**